tmexp/metrics.py

```diff
@@ -24,6 +24,21 @@ def metric_stats(metric: np.array, num_tabs: int, logger: Logger) -> None:
         logger.info("%s%s : %.2f", num_tabs * "\t", op_name, op(metric))
 
 
+def compute_distinctness(
+    wordtopic: np.array, num_topics: int, num_words: int
+) -> np.array:
+    distinctness = np.zeros((num_topics, num_topics))
+    wordtopic += 1 / num_words
+    wordtopic /= np.sum(wordtopic, axis=1)[:, None]
+    for i in range(num_topics):
```

Comment:
```suggestion
    for dist_i in wordtopic:
```

Reply:
done

Reply:
ah actuaally i need the i index after, will do enumerate

Reply:
but you need it to get dist_i right? Or did I miss something?

Reply:
but you need it to get dist_i right? So don't need it anymore no?

Reply:
below I also need it to put it in the distinctness matrix